### ISSUE TITLE
Redirect 30000000000000004.com

### ISSUE THREAD

**Issue by MusabShakeel576** [erikwiffin/0.30000000000000004#207]
Hi there,
It would be good to redirect people who visited 30000000000000004.com to 0.30000000000000004.com

**Comment by hoosierEE** [comment]
Are there programming languages where `10000000000000000 + 20000000000000000 = 30000000000000004`?

**Comment by lts20050703** [comment]
I don't think it would be good to redirect, but if you have already bought the domain might as well as use it as the homepage or something like "You might be looking for 0.30000000000000004.com"
